include/browsers/mswebview2/Ms_Webview2.hh

```diff
@@ -97,6 +97,7 @@ private:
   /// UI changes and then natively pass them to pseudo-children.
   ///
   /// For this purpose, We hook into the GTK event loop with `Glib::signal_idle`

```

Comment:
@Zacharymk1213
 `layout_update_cb` should become a pure virtual class in `gbw::gtk::Window` and then OS/browser specific logic should inherit hierarchically. I just initially implement this here for MsWebview2 as a quick and dirty and have not yet got back to making it portable.